test/common/common/assert_test.cc

```diff
@@ -83,10 +83,11 @@ TEST(EnvoyBugDeathTest, VariousLogs) {
   EXPECT_ENVOY_BUG({ ENVOY_BUG(false, ""); }, "envoy bug failure: false.");
   EXPECT_ENVOY_BUG({ ENVOY_BUG(false, "With some logs"); },
                    "envoy bug failure: false. Details: With some logs");
+  EXPECT_ENVOY_BUG({ ENVOY_BUG(false, ""); }, "stacktrace for envoy bug");
```

Comment:
Can we have stack trace being logged in the test? The above message would be logged even if the rest of the flow failed

Reply:
Done. Added another test to check if testing::Test::Run() is in the stack